xla/service/gpu/gpu_compiler.cc

```diff
@@ -2189,66 +2189,44 @@ absl::Status GpuCompiler::RunPostSchedulingPipelines(
     int64_t offloading_memory_limit_gb =
         module->config().debug_options().xla_gpu_offloading_memory_limit_gb() *
         1024 * 1024 * 1024;
-    float offloading_bandwidth =
-        module->config()
-            .debug_options()
-            .xla_gpu_offloading_bandwidth_bytes_per_sec();
```

Comment:
Can you also infer flops_per_sec ?
gpu_device_info has fpus_per_core and other function like clock_rate_ghz, that maybe allow you to infer it.

Reply:
Let me figure it out.

Reply:
Modified and verified with a V100 locally.